### PR TITLE
Webpage

### PR DESCRIPTION
**Pull request by stereobooster**
screen shot 2017-11-16 at 21 10 40

Demo https://stereobooster.github.io/css-in-js/

We can get domain from https://js.org/

### CODE REVIEW

**Comment by MicheleBertoli** [comment]
Thanks @stereobooster for your PR.
@rofrischmann was working on a web page for the repo a while ago but we never had time to finish it.

**Comment by will-stone** [comment]
This is great. May I suggest using Bootstrap for instant CSS boost to the table? Also a filtering mechanism would be good. For example, I only want to see those that support X

**Comment by stereobooster** [comment]
@will-stone yep you right bootstrap is better. 

Get again back to css-in-js dilemma, so need this table as decision tool. WDYT?

**Comment by stereobooster** [comment]
Ouch I confused branches

**Comment by stereobooster** [comment]
Ratio:
- it is easier to work with JSON, rather than markdown table. If you want I can generate markdown table from JSON file
- JSON file can have much more columns than markdown table
- web page can have advanced feature over readme in github, like filter based on features

File: src/components/Filter.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const Filter = options => {
```
Comment (MicheleBertoli) [inline src/components/Filter.js]: nit: rename `options` to `props`.

**Comment by MicheleBertoli** [comment]
Thank you very much @stereobooster, this looks pretty cool.
I like that you used `create-react-app`, and the code is simple and effective.

I agree it would make sense to have a single source of truth, and it would be great to generate the table from the same JSON file.
Also, as you said, the JSON file could have more columns (which is one of the most requested features for the repo).

I would love to have @rofrischmann's opinion on this.

**Comment by stereobooster** [comment]
I fixed your comment. Anything else you want me to do in this PR?

**Comment by rofrischmann** [comment]
Is the above example the newest version? I really like how simple it is! Would love to add more details later of course, but for now it's fine.
Just one thing: Can we maybe have a fixed width for the package column? It "jumps" around as I filter, which makes it hard to compare results visually.

**Comment by stereobooster** [comment]
Yes above generated from latest readme. There is a script which converts markdown to json

**Comment by stereobooster** [comment]
Made first two columns fixed width

**Comment by stereobooster** [comment]
Started small vocabulary https://github.com/stereobooster/css-in-js-systematic-approach, want to put definitions for all collumns (aka features) there. Comments are welcome

**Comment by MicheleBertoli** [comment]
You are awesome @stereobooster, thank you very much!
http://michelebertoli.github.io/css-in-js/

**Pull request merged**